lib/kennel/syncer.rb

```diff
@@ -133,7 +152,11 @@ def calculate_diff
           id = a.fetch(:id)
           if e
             diff = e.diff(a) # slow ...
-            @update << [id, e, a, diff] if diff.any?
+            if diff.any?
+              @update << [id, e, a, diff]
+            else
+              @already_ok << [id, e, a]
```

Comment:
better name ?
```suggestion
              @unchanged << [id, e, a]
```

Reply:
eehhhhhh not quite sure. See my long comment elsewhere about how I think of plan vs diff.

Reply:
Went with "no_change"